src/element/textWysiwyg.tsx

```diff
@@ -488,18 +488,21 @@ export const textWysiwyg = ({
     // in that same tick.
     setTimeout(() => {
       editable.onblur = handleSubmit;
-      // case: clicking on the same property → no change → no update → no focus
-      editable.focus();
```

Comment:
this one still needs to be there. Does the fix not work while keeping this?

Reply:
No the editor gets focussed so typing gets reflected on editor instead, I am not sure why do we need this ?

Reply:
When you click on an already selected button (e.g. you double-click on the same font button, or you open & close the color picker) there's no update which would trigger programmatic focus:

https://github.com/excalidraw/excalidraw/blob/c2a8712593a30de5c5785c44a08e0fd1eb7f0fce/src/element/textWysiwyg.tsx#L518-L521

Don't recall why the `pointerdown` preventing on the island isn't enough.